feature_extraction/bag_of_words.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
I am reluctant to create a new directory unless we know of a lot of algorythms that we want to add to it.  Would this fit in one of the directories that already exists?

Reply:
Feature extraction algorithms are
There are a lot of feature extraction algorithm. They usually go with machine learning.
Would `machine_learning/feature_extraction` be acceptable?